src/MOI_wrapper.jl

```diff
@@ -0,0 +1,414 @@
+using MathOptInterface
+const MOI = MathOptInterface
+const MOIU = MOI.Utilities
+
+# indices
+const VI = MOI.VariableIndex
+const CI = MOI.ConstraintIndex
+# supported functions
+const SVF = MOI.SingleVariable
+const SAF = MOI.ScalarAffineFunction{Float64}
+# supported sets
+const EQS = MOI.EqualTo{Float64}
+const GTS = MOI.GreaterThan{Float64}
+const LTS = MOI.LessThan{Float64}
+const INS = MOI.Interval{Float64}
+const BOUNDS = Union{EQS, GTS, LTS, INS}
+const BINS = MOI.ZeroOne
+const INTS = MOI.Integer
+const TYPES = Union{BINS, INTS}
+# support changes
+const SCC = MOI.ScalarCoefficientChange{Float64}
+# other MOI types
+const SAT = MOI.ScalarAffineTerm{Float64}
```

Comment:
`SAT` is an overloaded term, potentially a bit confusing to use as an alias. Generally aliases like these should be used in order to make the code more readable and not to save on typing.

Reply:
OK, many of these aliases were only used on one or two places, so I removed several of them.